dayrepo/snippets/templates/snippet_list.html

```diff
@@ -6,6 +6,9 @@ <h2>未提出日報一覧</h2>
     <p id="checklist">{{ checklist }}</p>
     <button onclick="location.href='{% url 'checklist_edit' checklist.id %}'">点検編集</button>
     <button onclick="location.href='{% url 'snippet_post' checklist.id %}'">snippet作成</button>
+    <form method="post" action={% url 'db_delete' checklist.id 'checklist' %}>
+        {% csrf_token %}
+        <button type="submit">点検削除</button>
```

Comment:
form タグ閉じてなくないです？
バグの元なので気をつけましょう〜

Auto Close Tagなどの  VScode 拡張機能はおすすめです
https://saishono.blog/setting-close-tag/

```suggestion
    <form method="post" action={% url 'db_delete' checklist.id 'checklist' %}>
        {% csrf_token %}
        <button type="submit">点検削除</button>
    </form>
```